### PR TITLE
Add additional git config option to foreman_templates

### PR DESCRIPTION
**Pull request by sean797**

### CODE REVIEW

File: plugins/foreman_templates/5.0/index.md [hunk 1]
```diff
@@ -108,6 +108,9 @@ To import templates back, after their content was changed, simply run
  
  If the templates are locked, changes won't be made. To update locked templates, either add `-d '{ "force": "true" }'` to the end of a command or unlock the template.
 
+## 3.2 Advanced git configuration
+
+Additional git configuration for foreman_templetes can be configured using the `git config` command line tool or editing the global/foreman user specific `.gitconfig` file. This can be useful when using a git server with a self signed cert, for a complete list of advanced options see `man git config`.
```
Comment (ares) [inline plugins/foreman_templates/5.0/index.md]: s/templetes/templates/

**Review by ares — CHANGES_REQUESTED** [review]
Thanks @sean797, this is a great addition. Would you mind providing an example of setting `http.sslCAPath` e.g. in foreman's config (or using `sudo -u foreman git config`). I think that would also close https://github.com/theforeman/foreman_templates/issues/69 :-)

**Comment by sean797** [comment]
Thanks for reviewing @ares. Updated

**Comment by ares** [comment]
Awesome, thanks @sean797, tested and works fine. Merging.

**Review by ares — APPROVED** [review]

**Pull request merged**